source/elements/oneDPL/source/parallel_api/parallel_range_api.rst

```diff
@@ -23,7 +23,15 @@ The following differences to the standard C++ range algorithms apply:
 - The execution policy parameter is added.
 - Output data sequences are defined as ranges, not iterators.
 - Both input and output ranges must support random access.
-- For a given algorithm, at least one of the input ranges as well as the output range must be bounded.
+- As a rule, both input and output ranges must be sized.
+
+  - Exceptions are ``transform``, ``equal``, and ``mismatch``, where only one of the input ranges must be sized,
```

Comment:
Do we need to clarify that we are talking about the "binary" overload of transform specifically?
It can be inferred from context, but perhaps its worth noting.

Reply:
Noted.